### PR TITLE
Add current pid

### PR DESCRIPTION
**Pull request by soyuka**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -9,15 +9,17 @@ module.exports = function (pid, cb) {
       return p.PID
     })
 
-    pidusage.stat(pids, function(err, stats) {
+    pids.push(pid)
```
Comment (soyuka) [inline index.js:12]: does this make sense @simonepri ?

Comment (simonepri) [reply index.js:12]: Absolutely I was thinking about that.
But i would push it on the top instead.

Comment (soyuka) [reply index.js:12]: ~~you mean unshift?~~

In fact maybe that psTree should return it.

Comment (simonepri) [reply index.js:12]: Yes I mean unshift. Anyway let me finish the work on pidusage.
Then I will dive on that.

**Comment by simonepri** [comment]
We can now use `pidtree` package which has the `root` option

**Comment by soyuka** [comment]
That's indeed what I want to do, opened an issue for that :).

**Pull request closed without merging**